$OpenBSD: patch-src_settings_cpp,v 1.1 2011/07/26 10:53:18 dcoppa Exp $

Disable set_window_title by default as it causes troubles
with wscons

--- src/settings.cpp.orig	Tue Jul 26 11:49:54 2011
+++ src/settings.cpp	Tue Jul 26 11:49:31 2011
@@ -428,7 +428,7 @@ void NcmpcppConfig::SetDefaults()
 	media_library_disable_two_column_mode = false;
 	discard_colors_if_item_is_selected = true;
 	store_lyrics_in_song_dir = false;
-	set_window_title = true;
+	set_window_title = false;
 	mpd_port = 6600;
 	mpd_connection_timeout = 15;
 	crossfade_time = 5;
